lib/core/testing/common/e2e_test_abstraction.dart

```diff
@@ -10,6 +11,7 @@ class E2ETestAbstraction {
     IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
     setUpAll(() async {
+      await EnvironmentConfig.inite2e();
```

Comment:
since that at the moment E2E doesn't works with docker container we need avoid call the local env